packages/gatsby-source-graphcms/README.md

```diff
@@ -0,0 +1,48 @@
+# gatsby-source-graphql
+
+Source plugin for pulling data into [Gatsby](https://github.com/gatsbyjs) from a GraphQL endpoint.
+
+## How to use
+*In your gatsby config...*
+```javascript
+plugins: [
+  /*
+   * Gatsby's data processing layer begins with “source”
+   * plugins. Here the site sources its data from a graphql endpoint
+   */
+  {
+    resolve: `gatsby-source-graphcms`,
+    options: {
+      endpoint: `graphql_endpoint`,
+      token: `graphql_token`,
+    },
+  }
+],
+```
+I suggest either using a `.env` file or setting environment variables to access ala `process.env.GATSBY_GRAPHQL_ENDPOINT`, `process.env.GATSBY_GRAPHQL_TOKEN` so theres no endpoint and token on git ;)
```

Comment:
Linking to https://www.gatsbyjs.org/docs/environment-variables/ would be better.

That page needs updated to include a reference to using `process.env` for plugins.

Also you don't want to use "I" when writing docs as this is a community plugin so generally either just use command form "Use either a `.env` file or set environment variables directly…" or use "We". But "we" is rarely what you want.